src/DynamoCoreWpf/Controls/StartPage.xaml.cs

```diff
@@ -417,6 +433,15 @@ private void OnStartPageLoaded(object sender, RoutedEventArgs e)
 
             var id = Wpf.Interfaces.ResourceNames.StartPage.Image;
             StartPageLogo.Source = dynamoViewModel.BrandingResourceProvider.GetImageSource(id);
+
+            if (dynamoViewModel.Model.PreferenceSettings.IsFirstRun)
+            {
+                dynamoViewModel.Model.PreferenceSettings.IsFirstRun = false;
+                GalleryViewModel galleryViewModel = new GalleryViewModel(dynamoViewModel);
+                GalleryView galleryView = new GalleryView(galleryViewModel);
+                if (dynamoViewModel.ShowGalleryUICommand.CanExecute(galleryViewModel))
+                    dynamoViewModel.ShowGalleryUICommand.Execute(galleryView);
+            }
```

Comment:
Hi @elayabharath, I don't think we will show `What's New` item only when Dynamo first run, right? Say if I close Dynamo immediately after launching it for the first time due to wrong key press, or that I don't even see the `What's New` item the first time round, then the button is gone forever.